cli/pkg/kctrl/cmd/app/init/vendir.go

```diff
@@ -58,32 +56,17 @@ func (v *VendirStep) Interact() error {
 	switch currentFetchOptionSelected {
 	case FetchFromGithubRelease:
 		githubStep := NewGithubStep(v.ui, v.config)
-		err := Run(githubStep)
-		if err != nil {
-			return err
-		}
+		return Run(githubStep)
 	case FetchFromHelmRepo:
 		helmStep := NewHelmStep(v.ui, v.config)
 		return Run(helmStep)
 	case FetchFromGit:
 		gitStep := NewGitStep(v.ui, v.config)
-		err := Run(gitStep)
-		if err != nil {
-			return err
-		}
+		return Run(gitStep)
 	case FetchChartFromGit:
```

Comment:
These 2 cases seems similar. Maybe we can just use a comma, like `case FetchFromGit, FetchChartFromGit:`?

Reply:
Adopted